ui/client/components/Fields.js

```diff
@@ -11,21 +11,20 @@ import ValidationLabels from "./modals/ValidationLabels";
 class RawField extends React.Component {
   render() {
     const markedClass = this.props.isMarked(this.props.index) ? " marked" : "";
-    const index = this.props.index;
-    const field = this.props.field;
     const opacity = this.props.isDragging ? 0 : 1;
+    const {index, field, isComparison} = this.props
 
     const validators = [notEmptyValidator];
     return this.props.connectDropTarget(this.props.connectDragSource(
       <div className="node-row movable-row" style={{opacity}}>
         <img src={dragHandleIcon} />
         <div className={"node-value fieldName" + markedClass}>
-          <input className={allValid(validators, field.name) ? "node-input" : "node-input node-input-with-error"}
+          <input className={isComparison || allValid(validators, field.name) ? "node-input" : "node-input node-input-with-error"}
```

Comment:
maybe let's name it showValidation? We can reuse it in the future :)

Reply:
Done